test/onnx/test_models_onnxruntime.py

```diff
@@ -36,11 +39,15 @@ def exportTest(self, model, inputs, rtol=1e-2, atol=1e-7, opset_versions=None):
     for opset_version in opset_versions:
         self.opset_version = opset_version
         self.onnx_shape_inference = True
-        run_model_test(self, model, input_args=inputs, rtol=rtol, atol=atol)
+        test_onnx_common._run_model_test(
```

Comment:
_run_model_test can be public, since: it is in test, and it is used in other modules.